### ISSUE TITLE
Volume percentage text missing

### ISSUE THREAD

**Issue by laurensvalk** [ev3dev/lms2012-compat#36]
When changing the volume, the numeric value is missing, like this:

![volpct-missing](https://cloud.githubusercontent.com/assets/12326241/19781842/a1490b22-9c8b-11e6-93ea-09a157765903.png)

```
Linux ev3dev 4.4.24-ti-rt-r55-16-ev3dev-bb.org #1 SMP PREEMPT RT Sat Oct 15 18:57:48 CDT 2016 armv7l GNU/Linux
```

**Comment by dlech** [comment]
It this still and issue for you with v1.99.2? I cannot reproduce.

**Comment by laurensvalk** [comment]
Yes, still an issue on FW V1.99C with 4.9.0-ev3dev-bpo-stretch-r2-ev3-lms2012.

**Comment by dlech** [comment]
OK, it is broken for me now. Maybe it works in debug builds? :confused:

**Comment by dlech** [comment]
This turned out to be a much more interesting/important bug that one would think. It turns out the problem is caused by the "hardening" compiler options that are used when building a debian package. In particular, [-D_FORTIFY_SOURCE](https://wiki.debian.org/Hardening#DEB_BUILD_HARDENING_FORTIFY_.28gcc.2Fg.2B-.2B-_-D_FORTIFY_SOURCE.3D2.29). This also explains why I did not see the problem in local builds.

**Comment by dlech** [comment]
Fix has been released in `lms2012-compat 1.99.3`. This also means that any number of the other open bugs could be fixed.
